lib/elastic_apm/trace_context/tracestate.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module ElasticAPM
+  class TraceContext
+    # @api private
+    class Tracestate
+      def initialize(values = [])
+        @values = values
+      end
+
+      attr_accessor :values
+
+      def self.parse(header)
+        # HTTP allows multiple headers with the same name, eg. multiple
+        # Set-Cookie headers per response.
+        # Rack handles this by joining the headers under the same key, separated
+        # by newlines, see https://www.rubydoc.info/github/rack/rack/file/SPEC
+        new(String(header).split("\n"))
```

Comment:
just curious why you used `String(..)` when `to_s` seems more commonly used in Ruby?

Reply:
I think it comes from how I often do `Array(value)` on passed arguments if I want to make sure it's an array.

Instead of doing `ArgumentError`, just turn it into a list.

In this case I don't think there's a difference in how `String(...)` and `.to_s` work(?). Happy to change it if you think this looks weird.

Reply:
I don't think there's a difference. I don't mind either way, I was just wondering if there was a benefit I was missing.